templates/dob_permitissuance.yml

```diff
@@ -84,6 +84,8 @@ dataset:
       - NTANAME
     sql: |
       SELECT 
+          ROW_NUMBER() OVER(PARTITION BY jobnum ORDER BY SUBSTR(dobrundate, 7, 4)||SUBSTR(dobrundate, 1, 2)||SUBSTR(dobrundate, 4, 2) DESC) as gid,
```

Comment:
Add comment to what this statement is doing and why it's needed.

Reply:
checked this logic was removed below.